dimension_c: 20
research_altitude: 1.5
takeoff_altitude: 1
edge_tolerance: 0.5
research_enabled: true
aggressive_enabled: false
preventive_enabled: false
simulation: false
max_altitude: 3
min_step: 0.3
max_step: 2
has_reach_destination_threshold: 0.5